[INFO   ] Logger: Record log in /storage/emulated/0/Download/WhatsAppClone/WhatsAppClone/.kivy/logs/kivy_24-12-17_51.txt
[INFO   ] Kivy: v2.2.1
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.11.4 (main, Sep 30 2023, 10:54:38) [GCC 11.4.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Logger: Purge log fired. Processing...
[INFO   ] Logger: Purge finished!
[INFO   ] KivyMD: 1.2.0, git-Unknown, 2024-08-06 (installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivymd/__init__.py")
[WARNING] KivyMD: Version 1.2.0 is deprecated and is no longer supported. Use KivyMD version 2.0.0 from the master branch (pip install https://github.com/kivymd/KivyMD/archive/master.zip)
[INFO   ] Factory: 190 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 v1.r32p1-01eac0.52688e0652bc37eab85ce86c674ab12f'>
[INFO   ] GL: OpenGL vendor <b'ARM'>
[INFO   ] GL: OpenGL renderer <b'Mali-G52 MC2'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <16383>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Clipboard: Provider: android
[ERROR  ] 
Traceback (most recent call last):
  File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/parser.py", line 553, in execute_directives
    mod = importlib.__import__(package)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1283, in __import__
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1126, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'kivy.uix.list'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/parser.py", line 556, in execute_directives
    mod = importlib.__import__(module_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1283, in __import__
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'kivy.uix.list'
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/parser.py", line 553, in execute_directives
[WARNING] stderr:     mod = importlib.__import__(package)
[WARNING] stderr:           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[WARNING] stderr:   File "<frozen importlib._bootstrap>", line 1283, in __import__
[WARNING] stderr:   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
[WARNING] stderr:   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
[WARNING] stderr:   File "<frozen importlib._bootstrap>", line 1126, in _find_and_load_unlocked
[WARNING] stderr:   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
[WARNING] stderr:   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
[WARNING] stderr:   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
[WARNING] stderr:   File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
[WARNING] stderr: ModuleNotFoundError: No module named 'kivy.uix.list'
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/parser.py", line 556, in execute_directives
[WARNING] stderr:     mod = importlib.__import__(module_name)
[WARNING] stderr:           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[WARNING] stderr:   File "<frozen importlib._bootstrap>", line 1283, in __import__
[WARNING] stderr:   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
[WARNING] stderr:   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
[WARNING] stderr:   File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
[WARNING] stderr: ModuleNotFoundError: No module named 'kivy.uix.list'
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/Download/WhatsAppClone/WhatsAppClone/main.py", line 517, in <module>
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/app.py", line 955, in run
[WARNING] stderr:     self._run_prepare()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/app.py", line 925, in _run_prepare
[WARNING] stderr:     root = self.build()
[WARNING] stderr:            ^^^^^^^^^^^^
[WARNING] stderr:   File "/storage/emulated/0/Download/WhatsAppClone/WhatsAppClone/main.py", line 462, in build
[WARNING] stderr:     print(args)
[WARNING] stderr:            ^^^^^
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 372, in load_string
[WARNING] stderr:     parser = Parser(content=string, filename=fn)
[WARNING] stderr:              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/parser.py", line 483, in __init__
[WARNING] stderr:     self.parse(content)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/parser.py", line 590, in parse
[WARNING] stderr:     self.execute_directives()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/parser.py", line 565, in execute_directives
[WARNING] stderr:     raise ParserException(self, ln,
[WARNING] stderr: kivy.lang.parser.ParserException: Parser: File "<inline>", line 3:
[WARNING] stderr: ...
[WARNING] stderr:       1:
[WARNING] stderr:       2:#: import Window kivy.core.window.Window
[WARNING] stderr: >>    3:#: import IRightBodyTouch kivy.uix.list.IRightBodyTouch
[WARNING] stderr:       4:#: import SliverTopAppBar __main__.SliverTopAppBar
[WARNING] stderr:       5:<CustomImage@ButtonBehavior+FitImage>:
[WARNING] stderr: ...
[WARNING] stderr: Unable to import package 'kivy.uix.list.IRightBodyTouch'
